src/store/Configuration/mutations.ts

```diff
@@ -0,0 +1,76 @@
+import { ConfigurationStateIF } from '@/interfaces'
+import axios from '@/axios-auth'
+
+export default {
+  setConfiguration (state: ConfigurationStateIF, data: any) {
+    state.configuration = data
+  },
+  setSessionVariables (state: ConfigurationStateIF, responseData: any) {
+    const authWebUrl: string = responseData['AUTH_WEB_URL']
+    sessionStorage.setItem('AUTH_WEB_URL', authWebUrl)
```

Comment:
We should have a look at what needs to be in session storage and what doesn't.

- is the variable needed for a different UI in the same domain?
- can the variable be fetched from the store when it is needed?
- does the variable need to be shared with another session (tab) in the same browser?
- does/should the variable need to survive if a session is closed and then reopened/restored?

Reply:
Those are really great questions Severin!  I'll answer your quesions point-by-point below:
 - my understanding was that filing-ui, edit-ui and create-ui each load a similar configuration file on start, so I wasn't thinking that session variables needed to be shared across UIs.  
 - Yes, the store is loaded early so I think the configuration can always be accessed via the store.  The only exceptions appear to be the "VUE_ROUTER_BASE" and "BASE_URL" which I haven't moved to the store.
 - my understanding is that if someone opens a new tab in the same browser, the configuration is loaded again as main.ts and App.vue would need to be called.
 - my assumption is that the configuration is immutable once it's loaded, if that's not the case please let me know I'll do further testing.

Reply:
OK, totally satisfactory answers, but I was more wondering if we need to save any of these values in session storage? Maybe not. As for being useful during development, well I think the Vue browser extension handles that nicely.

Reply:
I've done a search for `setItem(` and `getItem(` and I'm pretty sure I have a good handle on it.  But, refactoring always carries some risk especially when you're new on the project and don't know what you don't know.  :-)

Reply:
So... do we need any of these in session storage?

Reply:
No, not as far as I know

Reply:
Sweet. Nuke them?

Reply:
Yes, that's what I've done.  The only exceptions are the session variables related to routing.  For example:
- BUSINESS_ID
- TEMP_REG_NUMBER
- BASE_URL
- VUE_ROUTER_BASE

We need to keep these around.

Reply:
OK.